### ISSUE TITLE
Format C++ files

### ISSUE THREAD

**Issue by ironmig** [uf-mil/NaviGator#227]
Our CI software checks [for C++ compliance with the ROS C++ format standard](https://github.com/uf-mil/NaviGator/blob/kinetic-upgrade/Jenkinsfile). It should be possible to auto format all of our C++ files using these commands.
```
wget -O ~/.clang-format https://raw.githubusercontent.com/uf-mil/installer/master/.clang-format
clang-format-3.8 -stype=file -i <NAVIGATOR SOURCE FOLDER>
```

**Comment by ironmig** [comment]
Going for this for the easy green +'s

**Comment by ironmig** [comment]
Addressed in #228
